SU2_CFD/src/iteration_structure.cpp

```diff
@@ -2670,7 +2581,7 @@ void CDiscAdjFEAIteration::SetRecording(COutput *output,
 
     /*--- Clear indices of coupling variables ---*/
 
-    SetDependencies(solver, geometry, numerics, config, val_iZone, val_iInst, ALL_VARIABLES);
+    SetDependencies(solver, geometry, numerics, config, val_iZone, val_iInst, COMBINED);
```

Comment:
Personal preference: COMBINED is a bit unclear for me, since I don't know what you're combining just by looking at it.. if it's not too long, I would suggest explicitly naming the combined vars

Reply:
There are only two variables to combine now, solution and geometry, we also had the ALL_VARIABLES enum option, should I use that instead?

Reply:
I would prefer that, or feel free to make a new one SOL_AND_GEO (unless the number of these options is going to become very large and this type of naming would become problematic)